.env.template

```diff
@@ -0,0 +1,33 @@
+# Duplicate this file in your local repo with filename `.env`. It will be ignored by git, but loaded
+# by Gradle via a dotenv Gradle plugin and passed to Toolbox to use as instance configs.
+
+# Note that any values provided in this template will never be read - they are there to provide
+# example values only. Any uncommented vars in this file *must* be set in a local .env file.
+# See the README for more information.
+
+# Required variables - must be set in .env, build will throw if missing.
+APP_TOOLBOX_ENVIRONMENT=Development
+APP_TOOLBOX_DB_HOST=localhost
+APP_TOOLBOX_DB_SCHEMA=toolbox
+APP_TOOLBOX_DB_USER=toolbox
+APP_TOOLBOX_DB_PASSWORD=toolbox
```

Comment:
As per our discussion in our team meeting, could remove these defaults from the template, as these values will not be used (as the comment at top explains).  

I could go either way.  They don't seem harmful to have here, and it's kind of nice to see a sample value (which might well be a working value).  That said, they also should not be difficult for a developer to work out what values go in those slots, at least for these vars.